packages/nx/src/utils/versions.ts

```diff
@@ -1,6 +1,6 @@
 export const nxVersion = require('../../package.json').version;
 
-export const angularCliVersion = '~15.1.0';
+export const angularCliVersion = '~15.2.0';
```

Comment:
Sidenote: are we ready to remove this yet?

Reply:
I do believe that it is unused within Nx package. I'll double check